### PR TITLE
{lib}[GCCcore/8.3.0] FSON v1.0.5

### PR DESCRIPTION
**Pull request by DirectorSloan**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/f/FSON/FSON-1.0.5-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+builddependencies = [
+    ('binutils', '2.32'),
+    ('CMake', '3.15.3'),
+]
+
+configopts = "-DBUILD_SHARED_LIBS=ON"
+
+sanity_check_paths = {
+    'files': ['lib64/libFSON.so'],
```
Comment (Micket) [inline easybuild/easyconfigs/f/FSON/FSON-1.0.5-GCCcore-8.3.0.eb:23]:
```suggestion
    'files': ['lib64/libFSON.%s' % SHLIB_EXT]
```

File: easybuild/easyconfigs/f/FSON/FSON-1.0.5-GCCcore-8.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+source_urls = ['https://github.com/josephalevin/fson/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['d52f928d59b090eb47ed2bc880ee58d46cd8dd7e554e3df03c7c517d7a6b6012']
+
+builddependencies = [
+    ('binutils', '2.32'),
+    ('CMake', '3.15.3'),
+]
+
+configopts = "-DBUILD_SHARED_LIBS=ON"
```
Comment (Micket) [inline easybuild/easyconfigs/f/FSON/FSON-1.0.5-GCCcore-8.3.0.eb:20]:
```suggestion
configopts = ['-DBUILD_SHARED_LIBS=OFF', '-DBUILD_SHARED_LIBS=ON']
```
usually nice to provide both (though this is not a requirement)

**Review by Micket — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@DirectorSloan Thanks a lot for your contribution! Are you up for making the requested changes?

**Comment by DirectorSloan** [comment]
will make a new one

**Pull request closed without merging**